### PR TITLE
Cloud Spanner Dialect announcement and tutorial

### PR DESCRIPTION
**Pull request by saturnism**

### CODE REVIEW

**Comment by Sanne** [comment]
thanks @saturnism ! Pushing it to staging, so we'll be able to see how it looks like, check formatting.

Assuming it looks fine, want me to publish it ASAP or need a specific date/time?

**Comment by Sanne** [comment]
https://staging.in.relation.to/

**Comment by Sanne** [comment]
ah some links are using the wrong format, fixing them...

**Comment by Sanne** [comment]
all ready, will publish soon?

File: posts/Ray/2019-12-17-google-cloud-spanner-dialect.adoc [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-hibernate-orm</artifactId>
+</dependency>
+----
+
+Add both the https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc[Cloud Spanner JDBC driver] and the https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate[Cloud Spanner Dialect] dependencies to the `pom.xml:
```
Comment (gunnarmorling) [inline posts/Ray/2019-12-17-google-cloud-spanner-dialect.adoc:93]: \`pom.xml -> \`pom.xml\`

**Comment by gunnarmorling** [comment]
Perhaps have a note at the top saying "This is a guest post of Ray Tsang of Google..."?

File: posts/Ray/2019-12-17-google-cloud-spanner-dialect.adoc (hunk 1 above)
Comment (Sanne) [reply posts/Ray/2019-12-17-google-cloud-spanner-dialect.adoc:93]: thanks man :)

**Comment by Sanne** [comment]
> Perhaps have a note at the top saying "This is a guest post of Ray Tsang of Google..."?

I'm not sure about that. Where to draw the line? Will we add one if you write something about DBZ, since you're no longer part of Hibernate yourself?  I'd rather see it as an open community of people interested in data. But thanks I'll ask what others think about this for the future.

**Comment by gunnarmorling** [comment]
> But thanks I'll ask what others think about this for the future.

Of course, just wanted to bring it up. For the Debezium blog we have this
note if it's non-Hatters blogging (which they are always invited to do).

**Comment by Sanne** [comment]
cool, I understand but my goal is to have many more people actively involved who are not part of RH :)

**Comment by Sanne** [comment]
published! many thanks @saturnism & @gunnarmorling , let's break the news now.

**Pull request merged**